### PR TITLE
Update 1password extension

### PR DESCRIPTION
**Pull request by bxb100**
## Description

image

## Screencast

origianl
image

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/1password/src/v8/utils.ts [hunk 1]
```diff
@@ -21,8 +22,11 @@ export class ExtensionError extends Error {
 }
 
 export class NotFoundError extends ExtensionError {}
+
 export class CommandLineMissingError extends ExtensionError {}
+
 export class ZshMissingError extends ExtensionError {}
+
 export class ConnectionError extends ExtensionError {}
```
Comment (LitoMore) [inline extensions/1password/src/v8/utils.ts:30]: Can you revert these unrelated changes? Let's keep the original author's code style and only modify the code relevant to the pull request.

If every contributor formats the code once according to their preferences, then git blame will become very bad.

**Review by LitoMore — CHANGES_REQUESTED** [review]

File: extensions/1password/src/v8/utils.ts (hunk 1 above)
Comment (bxb100) [reply extensions/1password/src/v8/utils.ts]: 👌 Thanks review

**Comment by LitoMore** [comment]
I don't encourage loading web resources in this security-sensitive extension, and I'm conservative about it to make sure it feels safe.

Of course, it would be a better choice if we could find a logic to let it only get icons from https://simpleicons.org.

**Comment by bxb100** [comment]
@LitoMore agree with you, so I choose to use Google server with domain info `const domain = hostname.split(".").slice(-2).join(".");`

and the community had using `https://icons.bitwarden.net/${domain}/icon.png` 
https://github.com/raycast/extensions/blob/cbd0ba64a05aac10c4687e7ee8ea8c7b7135cc26/extensions/bitwarden/src/utils/search.ts#L42

**Comment by bxb100** [comment]
@LitoMore find this https://support.1password.com/rich-icons-privacy/ , now I think it's not a good idea to introduce this feature, thanks review
